pkg/releaser/git.go

```diff
@@ -157,6 +163,47 @@ func (r *Releaser) submitPR(request *source.ReleaseRequest) (string, error) {
 	return pr.GetHTMLURL(), nil
 }
 
+func (r *Releaser) closeExistingPR(request *source.ReleaseRequest, client *github.Client) error {
+	queryString := fmt.Sprintf("is:pr is:open author:%s repo:%s/%s release new version %s",
+		request.PluginReleaseActor,
```

Comment:
this should be r.TokenUserHandle as the PR are opened by krew-release-bot user. author/actor is the plugin author.

Reply:
:heavy_check_mark: